src/scheduler/gc_work.rs

```diff
@@ -550,8 +561,11 @@ impl<E: ProcessEdgesWork> RootsWorkFactory for ProcessEdgesWorkRootsWorkFactory<
         );
     }
 
-    fn create_process_node_roots_work(&mut self, _nodes: Vec<ObjectReference>) {
-        todo!()
+    fn create_process_node_roots_work(&mut self, nodes: Vec<ObjectReference>) {
+        // We want to use E::create_scan_work.
```

Comment:
You can put an assertion here to check if the plan moves objects. So if a binding tries to use this method in a moving plan, they will get a panic immediately. 
https://github.com/mmtk/mmtk-core/blob/9a3ebff00d006388fb6369dd7c78a9b2cb74f647/src/plan/plan_constraints.rs#L11